cli/src/commands/describe.rs

```diff
@@ -195,7 +215,10 @@ pub(crate) fn cmd_describe(
     let commit_descriptions: HashMap<_, _> = commit_descriptions
         .into_iter()
         .filter_map(|(commit, new_description)| {
-            if *new_description == *commit.description() && !args.reset_author {
+            if *new_description == *commit.description()
+                && !args.reset_author
+                && args.author.is_none()
```

Comment:
nit: strictly speaking, we should test `new_author == commit.author()` because new author might be unchanged (unlike `--reset-author`.) Perhaps, we can refactor `commit_descriptions` to a list of `(old_commit, new_author, new_description)`.

This isn't a blocker. Can be addressed later.

Reply:
This is more complicated, we need to test whether the author is none or equal, so it's either:
```rust
                && !args.author.as_ref().is_some_and(|(name, email)| {
                    name != &commit.author().name || email != &commit.author().email
                })
```
or
```rust
                && args.author.is_none()
                && args.author.as_ref().is_some_and(|(name, email)| {
                    name == &commit.author().name && email == &commit.author().email
                })
```

Reply:
Yep, we might want to refactor it later.

You could use `.map_or(true, ..)`, btw, but it might be slightly simpler if we invert the condition.
```rust
        .filter(|(commit, new_description)| {
            new_description != commit.description()
                || args.reset_author
                || args.author.as_ref().is_some_and(|(name, email)| {
                    name != &commit.author().name || email != &commit.author().email
                })
        })
        .map(|(commit, new_description)| (commit.id(), new_description))
```

Reply:
> I'll consider how to refactor problematic code in describe.rs but probably not until next weekend.

Oops, I didn't mean this was a blocker. I think the current state is good, and this PR can be merged if you're okay.

Reply:
Oh, sorry. I have badly worded that sentence.
I mean I'll investigate it but likely not before next weekend, and I've enabled auto-merge. The last time I used GH auto merge, the conversations weren't a blocker, so I'll resolve them.